doc/03-Configuration.md

```diff
@@ -91,3 +98,19 @@ allowing to keep this information for longer with a smaller storage footprint.
 
 A duration string is a sequence of decimal numbers and a unit suffix, such as `"20s"`.
 Valid units are `"ms"`, `"s"`, `"m"` and `"h"`.
+
+### Galera Cluster
+
+Icinga DB might not function properly or even crash occasionally when it is used with [Galera cluster](https://mariadb.com/kb/en/what-is-mariadb-galera-cluster/) setups.
+To avoid such unexpected crashes and reading/writing inconsistent data, Icinga DB sets the [wsrep_sync_wait](https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_sync_wait)
+system variable for all its database connections. Consequently, strict cluster-wide causality checks are enforced before
+executing specific SQL queries, which are determined by the value set in the `"wsrep_sync_wait"` system variable. By default,
+Icinga DB sets this to `7`, which includes `READ, DELETE, INSERT, REPLACE` query types and is usually sufficient.
+Unfortunately, this also has the downside that every single Icinga DB query will be blocked until the cluster nodes
+resynchronise their states after each executed query, and may result in degraded performance.
+
+However, this does not necessarily have to be the case if, for instance, Icinga DB is only allowed to connect to a single
+cluster node at a time. This is the case when a load balancer does not randomly route connections to all the nodes evenly,
+but always to the same node until it fails, or if your database cluster does not implement an IP address fail over. In such
```

Comment:
> does not implement an IP address fail over

If a virtual IP address is being used, which is only changed for failover situations - e.g., by keepalived - no `wsrep_sync_wait` should be necessary. I would change this sentence to:

> […], or if your database cluster nodes have a virtual failover IP address assigned.